composer.json

```diff
@@ -141,7 +137,7 @@
         "drupal-composer/drupal-scaffold": "^2.0.0",
         "drupal/pathauto": "^8.1.0",
         "drupal/entity_browser": "8.1.0-alpha6",
-        "drupal/views_infinite_scroll": "8.1.0",
+        "drupal/views_infinite_scroll": "8.1.2",
```

Comment:
Can this be ^8.1.0? We don't use pinned versions unless we're applying a patch, and since this PR thankfully removes a patch, we no longer need to pin the version. :)